### PR TITLE
sync the image with latest updates.

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
Added a new ruby script to create a superuser based on user credentials.

#### things were added
- added a new env var to check if the user wants to log in via threefold connect
- change the base image from `mastodon` to `Emad image` (want to update to tf-image)
#### related issues
[#112-Update mastodon image, Support tf-connect](https://github.com/threefoldtech/tf-images/issues/112)

### CODE REVIEW

File: tfgrid3/mastodon/docker/docker-compose.yaml [hunk 1]
```diff
@@ -56,7 +55,7 @@ services:
 
   web:
     # build: .
-    image: tootsuite/mastodon:v4.0.0rc1
+    image: "mahmoudemmad/mastodon:tfconnect"
```
Comment (AhmedHanafy725) [inline tfgrid3/mastodon/docker/docker-compose.yaml]: please push this image to threefolddev account and use it

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: tfgrid3/mastodon/docker/docker-compose.yaml (hunk 1 above)
Comment (Mahmoud-Emad) [reply tfgrid3/mastodon/docker/docker-compose.yaml]: Updated

File: tfgrid3/mastodon/scripts/config.sh [hunk 2]
```diff
@@ -4,6 +4,11 @@ DOCKER_DIR=../docker/
 SCRIPTS_DIR=../scripts/
 ENV_DIR=${DOCKER_DIR}.env.production
 
+# Must send mastodon image to env vars when run the docker-compose.
+# OPTIONS:
+	# tootsuite/mastodon:v4.0.0rc1
+	# mahmoudemmad/mastodon:tfconnect
```
Comment (AhmedHanafy725) [inline tfgrid3/mastodon/scripts/config.sh]: still need to be updated

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: tfgrid3/mastodon/scripts/config.sh (hunk 2 above)
Comment (Mahmoud-Emad) [reply tfgrid3/mastodon/scripts/config.sh]: Updated, removed unused comment.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**